pkg/sql/opt/exec/execbuilder/testdata/catalog

```diff
@@ -267,3 +267,9 @@ TABLE abx
            ├── table=59 index=1 (1 key column)
            └── table=60 index=1 (1 key column)
 scan abx
+
+query T
```

Comment:
Please add a test in `logictest/testdata/pg_catalog` for doing a point lookup on an existing type and a user defined (enum) type

Reply:
Done